docs/pages/eas/hosting/custom-domain.mdx

```diff
@@ -44,7 +44,47 @@ You will need to own a domain name you want to use.
+> That is, add the **Verification TXT** record first, and press "Refresh" until the UI confirms the verification record. Then add the **SSL CNAME** record next until
+> it is confirmed, and set up the third record last.
 > If downtime isn't important or relevant, you may add all three DNS records at once.
 
 After assigning a custom domain to your app, the custom domain will route to your **production** deployment.
+
+### Custom domain DNS records
+
+Two of the three records the dashboard presents you are to validate ownership of your domain.
+The **Verification TXT** record proves ownership of your domain, since it adds a custom token that can be read back to verify you're setting the domain up on a domain you control.
+The **SSL CNAME** record proves ownership of your domain to a certificate authority, also known as Domain Control Validation (DCV). This is a CNAME record because both renewal and validation is delegated to an automated process, which prevents certificates from expiring.
+
+Both records are created on a subdomain of the custom domain you're setting up.
+
+- If you're setting up `example.com`, the records must be created on `_cf-custom-hostname.example.com` and `_acme-challenge.example.com` respectively
+- If you're setting up `anything.example.com`, the records must be created on `_cf-custom-hostname.anything.example.com` and `_acme-challenge.example.com` respectively
+
+Lastly, the third DNS entry that the dashboard presents will always be the actual DNS record that points your domain at EAS Hosting.
+
+- For apex domains the dashboard typically recommends an **A record** to `172.66.0.241`
+- For subdomains, the dashboard typically recommends a **CNAME record** to `fallback.expo.app`
+
+Both of these records are equivalent, however some DNS providers do not allow CNAME records to be set up on apex domains.
+
+### Alias and wildcard subdomains
+
+> **info** If you had a custom domains set up already before **19 March 2025**, you must press the "Refresh" button in your project's [Hosting settings](https://expo.dev/accounts/[accountName]/projects/[projectName]/hosting/settings) before following instructions for setting up wildcard domains.
+
+While only a single custom domain can be set up per project, you can set up further subdomain DNS records to handle requests for other aliases than just the production alias.
+As long as a subdomain record is set up with a label that matches an alias in your hosting project, requests will be handled by that alias' deployment.
+
+For example, after [creating a `staging` alias](/eas/hosting/deployments-and-aliases/#aliases), you may set up CNAME record for your alias:
+
+- If you've set up an apex domain, for example `example.com`, create a CNAME record on `staging.example.com` set to `fallback.expo.app`
+- If you've set up subdomain domain, for example `anything.example.com`, create a CNAME record on `staging.anything.example.com` set to `fallback.expo.app`
```

Comment:
I wonder if tables would make this clearer: domain type, CNAME record label, CNAME record value

Reply:
I've tried avoiding them so far since I'm not sure if they'll be "too precise" since the instructions will vary depending on the nameserver provider. But I'll have a browse to see what others are doing and will send a follow-up PR once I've got a better idea of what a standard table would look like.

Maybe the dig like output will do the trick, since we can also show people how to run `dig` to verify the DNS records